shaq/_cli.py

```diff
@@ -111,7 +111,8 @@ async def _shaq(console: Console, args: argparse.Namespace) -> dict[str, Any]:
         input = _from_file(console, args)
 
     shazam = Shazam(language="en-US", endpoint_country="US")
-    return await shazam.recognize(input)  # type: ignore
+
+    return await shazam.recognize_song(input, proxy=args.proxy)  # type: ignore
```

Comment:
Could you change this back to `recognize`? My understanding is that `recognize` also takes a `proxy` and is significantly faster than `recognize_song`, since it uses the Rust backend.

Reply:
Oh right, I've been sitting on this commit for a while and now I'm remembering that `recognize` wasn't working in my tests… but I'm failing to remember why. I think it was erroring for me for whatever reason? I need to try it again locally I guess.